### DISCUSSION TITLE
3.81.105.209 (Getting error while doing terraform plan)

### DISCUSSION THREAD

**Discussion by sravanreddy28** [learndevopsonline/learndevopsonline#1416] [Q&A] [answered]
Hi @r-devops,
Getting error while doing terraform plan
workstation IP: **3.81.105.209**
Image : https://github.com/learndevopsonline/learndevopsonline/assets/138639644/7304793e-d266-4a80-8900-5a4482e1aed9

Thanks,

**Comment by sreedharm07** [comment, marked as answer]
Hi @sravanreddy28,
Our  lab name is= Centos-8-DevOps-Practice
you have given = Centos-8-DevOps_Practice.  thats y its showing no results with that name
update it and try once. 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/2343442e-0343-4329-ae9b-59a08c0b3df7)
